tests.py

```diff
@@ -103,24 +107,29 @@ def test_get_sld_from_builtin_full_publicsuffix_org_list_with_mixed_case(self):
 
     def test_get_sld_from_builtin_full_publicsuffix_org_list_with_leading_dot(self):
         psl = publicsuffix.PublicSuffixList(None)
-        assert 'com' == psl.get_sld('.com')
-        assert 'example' == psl.get_sld('.example')
+        # '.com' -> <empty>.<com> -> None, empty labels are not allowed
+        assert None == psl.get_sld('.com')
+        # '.example' -> <empty>.<example> -> None, empty labels are not allowed
+        assert None == psl.get_sld('.example')
         assert 'example.com' == psl.get_sld('.example.com')
-        assert 'example' == psl.get_sld('.example.example')
+        # note: non-strict mode: TLD 'example' -> SLD example.example
+        assert 'example.example' == psl.get_sld('.example.example')
 
     def test_get_sld_from_builtin_full_publicsuffix_org_list_with_unlisted_tld(self):
         psl = publicsuffix.PublicSuffixList(None)
         assert 'example' == psl.get_sld('example')
-        assert 'example' == psl.get_sld('example.example')
-        assert 'example' == psl.get_sld('b.example.example')
-        assert 'example' == psl.get_sld('a.b.example.example')
+        # non-strict mode, tld=example, sld=example.example
+        assert 'example.example' == psl.get_sld('example.example')
+        assert 'example.example' == psl.get_sld('b.example.example')
+        assert 'example.example' == psl.get_sld('a.b.example.example')
 
     def test_get_sld_from_builtin_full_publicsuffix_org_list_with_listed_ut_non_internet_tld(self):
         psl = publicsuffix.PublicSuffixList(None)
         assert 'local' == psl.get_sld('local')
-        assert 'local' == psl.get_sld('example.local')
-        assert 'local' == psl.get_sld('b.example.local')
-        assert 'local' == psl.get_sld('a.b.example.local')
+        # note: non-strict mode, unknown tld: local -> sld: example.local
```

Comment:
my thoughts are that this whole suite might be removed. local is not listed in the psl anymore -- though i believe it was once and that's how the test was made. this series of tests assumes something is true that isn't in our control -- the psl.  i'm not sure what this unit does at all to help.